### ISSUE TITLE
Log_threshold vs threshold gradients

### ISSUE THREAD

**Issue by michaelttpearce** [bartbussmann/BatchTopK#2]
In the JumpReLUFunction class, the input parameter is log_threshold which is used to define the threshold as torch.exp(log_threshold). But the (pass-through) gradients computed seem to be for the threshold, not the log_threshold.
